saagieapi/jobs/gql_queries.py

```diff
@@ -82,50 +91,102 @@
         }
     }
 }
-  """
+"""
 
 GQL_GET_JOB_INSTANCE = """
 query jobInstanceQuery($jobInstanceId: UUID!){
     jobInstance(id: $jobInstanceId){
-      id
-      number
-      status
-      startTime
-      endTime
-      jobId
-      version {
+        id
         number
-        releaseNote
-        runtimeVersion
-        commandLine
-        isMajor
-        isCurrent
-      }
+        status
+        history {
```

Comment:
The doc for the section `Getting job instance information` should be updated with the new GQL query